tests/foreman/cli/test_errata.py

```diff
@@ -459,6 +459,36 @@ def test_negative_install_by_hc_name_without_org_info(self):
             )
         self.assertIn('Error: Could not find organization', context.exception.stderr)
 
+    @tier3
+    @upgrade
+    def test_positive_list_affected_chosts(self):
+        """View a list of affected content hosts for an erratum
+
+        :id: 3b592253-52c0-4165-9a48-ba55287e9ee9
+
+        :Setup: Errata synced on satellite server.
+
+        :Steps: host list --search "applicable_errata = <erratum_id>"
+            --organization-id=<org_id>
+
+        :expectedresults: List of affected content hosts for an erratum is
+            displayed.
+
+        :CaseAutomation: automated
+
+        """
+        result = Host.list(
+            {
+                'search': 'applicable_errata = {}'.format(FAKE_2_ERRATA_ID),
+                'organization-id': self.org['id'],
+                'fields': 'Name',
+            }
+        )
+        for virtual_machine in self.virtual_machines:
+            assert virtual_machine.hostname in str(
+                result
+            ), "VM host name not found in list of applicable hosts"
```

Comment:
That `str()` call doesn't sound right, as you might end up matching substring instead of exact string. I think it's better to get list of hostnames and assert your string is in that list:
```python
result = [item['name'] for item in result]
assert virtual_machine.hostname in result, "VM host name not found in list of applicable hosts"
```

Reply:
Thanks for that suggestion. Change made, test coming soon.